web/client-api/src/main/java/io/deephaven/web/client/api/widget/plot/JsFigure.java

```diff
@@ -201,6 +203,8 @@ public Promise<JsFigure> refetch() {
                     .map(chartDescriptor -> new JsChart(chartDescriptor, this)).toArray(JsChart[]::new);
             JsObject.freeze(charts);
 
+            errors = JsObject.freeze(descriptor.getErrorsList().asList().toArray(new String[0]));
```

Comment:
maybe just array.slice?
```suggestion
            errors = JsObject.freeze(descriptor.getErrorsList().slice());
```
Also have to change the type of the field and the getter